eng/common/scripts/Update-DevOps-Release-WorkItem.ps1

```diff
@@ -69,8 +71,12 @@ $plannedVersions = @(
     Date = $plannedDate
   }
 )
+$ignoreReleasePlannerTests = $true
+if ($tag -eq "Release Planner App Test") {
```

Comment:
tag could contain multiple values. would it be better to do contains instead of eq?

Reply:
When calling from release planner app test, I believe it will only be one tag that's why I checked for eq.

Reply:
Updated to use `Contains'

Reply:
For nwo that is true. I guess if it ever changes we will just have to update it here too